src/goto-harness/recursive_initialization.cpp

```diff
@@ -203,17 +208,23 @@ irep_idt recursive_initializationt::build_constructor(const exprt &expr)
   // void type_constructor_T(int depth_T, T *result_T, int *size);
   optionalt<irep_idt> size_var;
   optionalt<irep_idt> expr_name;
+  bool is_nullable = false;
+  bool has_size_param = false;
   if(expr.id() == ID_symbol)
   {
     expr_name = to_symbol_expr(expr).get_identifier();
+    is_nullable = initialization_config.potential_null_function_pointers.count(
+      expr_name.value());
```

Comment:
Maybe it's idiomatic, but I would prefer `count(..) != 0`

Reply:
I agree that'd be easier to read.